2.train-VAE/scripts/nbconverted/3.beta_vae_training_extract_model_components.py

```diff
@@ -55,10 +55,13 @@
 
 
 # subsetting the genes
-# create dataframe containing the 1000 genes with the largest variances and their corresponding gene label and extract the gene labels
-largest_var_df = gene_stats.nlargest(1000, "variance")
-gene_list = largest_var_df["gene_ID"].tolist()
-gene_list
+
+# create dataframe containing the genes that passed an initial QC (see Pan et al. 2022) and their corresponding gene label and extract the gene labels
+gene_dict_df = pd.read_csv("../0.data-download/data/CRISPR_gene_dictionary.tsv", delimiter='\t')
+gene_list = []
+for index, row in gene_dict_df.iterrows():
+    if row['qc_pass'] == True:
+        gene_list.append(row['dependency_column'])
```

Comment:
```suggestion
gene_list_passed_qc = gene_dict_df.query("qc_pass").dependency_column.tolist()
```

I think this is the same function, just a bit more pythonic

Reply:
Yep, adding this simplification